api/shape/shadow.js

```diff
@@ -4,9 +4,15 @@ import { cast as castGeometry } from '@jsxcad/geometry';
 
 export const shadow = Shape.registerMethod2(
   'shadow',
```

Comment:
Did this used to be called project? I feel like the name shadow might imply some sort of parallax where there is a point light source...although maybe I'm over thinking it and 'project' has the same issue

Reply:
I think it probably ought to be silhouette(). wdyt?

Reply:
Brilliant. That's exactly the right word